mandatory/3-function-output.js

```diff
@@ -1,17 +1,24 @@
 // Add comments to explain what this function does. You're meant to use Google!
+//The Math.random() function returns a floating-point, pseudo-random number that's greater than or equal to 0 and less than 1,
 function getRandomNumber() {
```

Comment:
I think here, the question is asking what does the `getRandomNumber()` does also?